services/listings/listings/archer.json

```diff
@@ -16,9 +12,28 @@
     "longitude": -121.91071
   },
   "applicationDueDate": "2019-12-31T15:22:57.000-07:00",
+  "applicationMethods": [
+    {
+      "type": "POBox",
+      "acceptsPostmarkedApplications": true
+    },
+    {
+      "type": "LeasingAgent"
+    },
+    {
+      "type": "PaperPickup"
+    }
+  ],
   "applicationOrganization": "98 Archer Street",
   "applicationPhone": "(408) 217-8562",
-  "blankPaperApplicationCanBePickedUp": true,
+  "assets": [
+    {
+      "referenceType": "Listing",
+      "referenceId": "Uvbk5qurpB2WI9V6WnNdH",
+      "label": "building",
+      "fileId": "https://regional-dahlia-staging.s3-us-west-1.amazonaws.com/listings/archer/archer-studios.jpg"
```

Comment:
pedantic, but perhaps call this `fileUrl` to make it more clear what it is?

Reply:
Well, eventually when we switch to a dedicated external file provider (Cloudinary maybe?), we'd only want to store the object ID of the file and the URL would be generated as a derivative of that.